GoogleMulticastAppDelegate.podspec

```diff
@@ -0,0 +1,72 @@
+Pod::Spec.new do |s|
+  # TODO: Is `GoogleMulticastAppDelegate` name fine?
+  s.name             = 'GoogleMulticastAppDelegate'
+  s.version          = '7.5.0'
+  s.summary          = 'GoogleMulticastAppDelegate'
+
+  s.description      = <<-DESC
+  GoogleMulticastAppDelegate
+                       DESC
+
+  s.homepage         = 'https://github.com/google/GoogleUtilities'
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+
+  s.source           = {
+    :git => 'https://github.com/google/GoogleUtilities.git',
+    :tag => 'MulticastAppDelegate-' + s.version.to_s
+  }
+
+  ios_deployment_target = '9.0'
+  osx_deployment_target = '10.12'
+  tvos_deployment_target = '10.0'
+  watchos_deployment_target = '6.0'
+
+  s.ios.deployment_target = ios_deployment_target
+  # s.osx.deployment_target = osx_deployment_target
+  # s.tvos.deployment_target = tvos_deployment_target
+  # s.watchos.deployment_target = watchos_deployment_target
+
+  s.swift_versions = ['5.0', '5.2']
```

Comment:
Delete now that this is Objective C

Reply:
After talking to Maksym, he thinks there should be no longer any blocker for making this SDK swift, let's discuss on Monday to see if it's still possible to implement this in swift.

Reply:
We're blocked on using Swift here until the next breaking change release, which is maybe OK?

We also need to confirm that it's ok for any 1P dependents to have a Swift dependency.